src/plugins/broadcaster.ts

```diff
@@ -0,0 +1,47 @@
+import { Plugin, Cordova } from './plugin';
+
+/**
+ * @name Broadcaster
+ * @description
+ * This plugin adds exchanging events between native code and your app.
+ *
+ * @usage
+ * ```
+ * import { Broadcaster } from 'ionic-native';
+ *
+ * // Listen to events from Native
+ * Broadcaster.addEventListener('eventName').then((event) => console.log(event));
+ *
+ * // Send event to Native
+ * Broadcaster.fireNativeEvent('eventName', {}).then(() => console.log('success'));
+ *
+ * ```
+ */
+@Plugin({
+  pluginName: 'Broadcaster',
+  plugin: 'cordova-plugin-broadcaster',
+  pluginRef: 'broadcaster',
+  repo: 'https://github.com/bsorrentino/cordova-broadcaster',
+  platforms: ['Android', 'iOS'],
+  install: 'ionic plugin add cordova-plugin-broadcaster'
```

Comment:
this line is not needed since we do not have a custom install command for this plugin